DigitalLearningSolutions.Data.Tests/Helpers/TutorialContentTestHelper.cs

```diff
@@ -44,5 +44,15 @@ public void UpdateDiagnosticAssessmentPath(int sectionId, string? newPath)
                 new { sectionId, newPath }
             );
         }
+
+        public void UpdateTutorialStatus(int tutorialId, int customisationId, int status)

```

Comment:
Yeah it was in the diagnostic helper for a brief time but it got removed. On a related note, most helpers are being used by more than one test class. Should we merge them together to have a unified set of helper methods for these tests? 🙂

Reply:
Good point 🙂 I think we want to avoid having one absolutely massive test helper. But we also don't need lots of little ones with no distinct individual responsibility. I'd be happy with:
* Merging them all into one learning menu test helper if it wouldn't be too big
* Merging them into a smaller set of helpers where each helper has a clear responsibility (e.g. one for tutorials, one for esctions).
* Keeping them as they are if each already has a clear responsibility
Either way if a change is needed then please do it as a separate PR 🙂

Reply:
Yeah the responsibility is very shared between the different helpers (eg a method to edit the status of a tutorial in the tutorial helper but one to edit as tutorial's diagnostic status in the section helper). I'll have a look at tidying this up after I'm done with this PR 🙂

Reply:
Great, thanks! 🙂